### DISCUSSION TITLE
part-ii/2009-122/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#77] [Question comments]
Part II 2009 Probability and Measure: Paper 3, Section II, J 

https://questions.tripos.org/part-ii/2009-122/

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning</summary>

Borel-Cantelli lemmas: The proof of the first is obvious. Remove one symbol and done. The proof of the second requires $1-a\leq e^{-a}$. 

Clearly the second Borel-Cantelli is useful. So the question boils down to a calculation of $$\sum_n \mathbb{P}\left(X\geq n^c\right)$$. This feels like a standard p-series calculation. 

I got $c = \frac{1}{2}$.
</details>
